bitcoin/src/address.rs

```diff
@@ -21,6 +21,12 @@
 //! // Generate pay-to-pubkey-hash address.
 //! let address = Address::p2pkh(&public_key, Network::Bitcoin);
 //! ```
+//!
+//! # Note: creating a new address requires the rand feature flag
+//!
+//! ```rust
+//! bitcoin = { version = "*", features = ["rand"] }
```

Comment:
Please, don't use wildcard here - it'd tempt people into using it in real code which is a bad idea. I suggest using `...`.
Also it's not `rust` but `toml`.